src/main/resources/wa-task-initiation-civil-generalapplication.dmn

```diff
@@ -2574,6 +2574,44 @@ else if (additionalData.Data.generalAppConsentOrder != null ) then true else fal
           <text>"generalApplicationsBeforeSdo"</text>
         </outputEntry>
       </rule>
+      <rule id="DecisionRule_1uwaglx">
+        <inputEntry id="UnaryTests_0l4pkh4">
+          <text>"REFER_TO_JUDGE"</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_07gnmux">
+          <text>"APPLICATION_SUBMITTED_AWAITING_JUDICIAL_DECISION"</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0blvr34">
+          <text>false</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_11bhqbw">
+          <text>true</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1luo1zu">
+          <text>true</text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1rfdb47">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_0r52bwp">
+          <text></text>
+        </inputEntry>
+        <inputEntry id="UnaryTests_1l7veu4">
+          <text>-</text>
+        </inputEntry>
+        <outputEntry id="LiteralExpression_09qcjno">
+          <text>"JudgeDecideOnApplication"</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_04vm90k">
+          <text>"Application for " + lower case(replace(additionalData.Data.generalAppType.types[1], "_"," "))</text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_1nthp1p">
+          <text></text>
+        </outputEntry>
+        <outputEntry id="LiteralExpression_17px1mv">
+          <text>"generalApplicationsBeforeSdo"</text>
```

Comment:
processCategoryIdentifier should be changed to generalApplications for after sdo